doc_source/serverless-clients.md

```diff
@@ -208,179 +215,292 @@ The following sample code uses the [opensearch\-js](https://www.npmjs.com/packag
+    console.log(response.body);
+    console.log((await client.indices.delete({ index })).body);
 }
 
-index_document().catch(console.log);
+main();
 ```
 
 ## Java<a name="serverless-java"></a>
 
-To ingest data into an OpenSearch Serverless with Java, use [AmazonOpenSearchJavaClient](https://github.com/awsdocs/amazon-opensearch-service-developer-guide/tree/master/sample_code/AmazonOpenSearchJavaClient-main), which is similar to the existing OpenSearch low\-level Java REST client\. The sample code in the repository establishes a secure connection to the specified collection, creates an index, and indexes a single document\. You must provide values for `region` and `host`\.
+The following sample code uses the [opensearch\-java](https://search.maven.org/artifact/org.opensearch.client/opensearch-java) client for Java to establish a secure connection to the specified OpenSearch Serverless collection and create a single index\. You must provide values for `region` and `host`\.
+
+The important difference compared to OpenSearch Service *domains* is the service name \(`aoss` instead of `es`\)\.
+
+```java
+SdkHttpClient httpClient = ApacheHttpClient.builder().build();
+
+OpenSearchClient client = new OpenSearchClient(
+    new AwsSdk2Transport(
+        httpClient,
+        "search-...us-west-2.es.amazonaws.com", // OpenSearch Serverless collection endpoint, without https://
+        "aoss" // signing service name
+        Region.US_WEST_2, // signing service region
+        AwsSdk2TransportOptions.builder().build()
+    )
+);
+
+String index = "sample-index";
+CreateIndexRequest createIndexRequest = new CreateIndexRequest.Builder().index(index).build();
+CreateIndexResponse createIndexResponse = client.indices().create(createIndexRequest);
+System.out.println("Create index reponse: " + createIndexResponse);
+
+DeleteIndexRequest deleteIndexRequest = new DeleteRequest.Builder().index(index).build();
+DeleteIndexResponse deleteIndexResponse = client.indices().delete(deleteIndexRequest);
+System.out.println("Delete index reponse: " + deleteIndexResponse);
+
+httpClient.close();
+```
 
 ## Python<a name="serverless-python"></a>
 
 The following sample code uses the [opensearch\-py](https://pypi.org/project/opensearch-py/) client for Python to establish a secure connection to the specified OpenSearch Serverless collection and create a single index\. You must provide values for `region` and `host`\.
 
 The important difference compared to OpenSearch Service *domains* is the service name \(`aoss` instead of `es`\)\.
 
-```
-from opensearchpy import OpenSearch, RequestsHttpConnection
-from requests_aws4auth import AWS4Auth
+```python
+from opensearchpy import OpenSearch, RequestsHttpConnection, AWSV4SignerAuth
 import boto3
 
-host = '' # The collection endpoint without https://. For example, 07tjusf2h91cunochc.us-east-1.aoss.amazonaws.com
-region = '' # e.g. us-east-1
-
+host = '' # cluster endpoint, for example: my-test-domain.us-east-1.es.amazonaws.com
```

Comment:
This needs to be "collection", as these docs are specific to OpenSearch Serverless